### PR TITLE
PD-2739, added Google Search Engine Result Pages (SERP) json-ld

### PR DESCRIPTION
**Pull request by huy-nguyen-colorado**
Purpose: Add Google SERP, JIRA task PD-2739

To test:

1.  Switch to v2.

2. Go to www.my.jobs and/or veterans.jobs, open browser web developer tool, please ensure this code is in the head section: 

Should look like this:

my_dot_jobs

veterans_dot_jobs

### CODE REVIEW

File: templates/home_page/home_page_listing_bootstrap3.html [hunk 1]
```diff
@@ -9,6 +9,21 @@
 
 {% endblock direct_extraHeaderContent %}
 
+{% block meta_serp_schema %}
+    <script type="application/ld+json">
+        {
+            "@context": "http://schema.org",
+            "@type": "WebSite",
+            "url": "http://www.indianapolis.jobs/",
```
Comment (koonsies) [inline templates/home_page/home_page_listing_bootstrap3.html]: What is the significance of this url being hardcoded to indianapolis.jobs?

Comment (haliscooper) [inline templates/home_page/home_page_listing_bootstrap3.html]: That would be bad.  The ticket says
"Where the URL and Target must be driven by which site the user is on."

File: templates/home_page/home_page_listing_bootstrap3.html [hunk 2]
```diff
@@ -9,6 +9,21 @@
...
 
+{% block meta_serp_schema %}
+    <script type="application/ld+json">
+        {
+            "@context": "http://schema.org",
+            "@type": "WebSite",
+            "url": "{{request.get_host}}",
+            "potentialAction": {
+                "@type":"SearchAction",
+                "target":"{{request.get_host}}/jobs/?q={q}&de_n=google&de_m=web&de_c=sitelinks",
```
Comment (tdphillips) [inline templates/home_page/home_page_listing_bootstrap3.html]: Is q a template variable or will it be interpreted by whatever makes use of this script?

If it's a Django template variable, the brackets need to be doubled.

Comment (huy-nguyen-colorado) [inline templates/home_page/home_page_listing_bootstrap3.html]: It's for the script, it acts as sor of a placeholder.

**Pull request merged**